central/complianceoperator/v2/checkresults/datastore/datastore_impl.go

```diff
@@ -57,6 +57,11 @@ func (d *datastoreImpl) SearchComplianceCheckResults(ctx context.Context, query
 	return d.store.GetByQuery(ctx, query)
 }
 
+// GetComplianceCheckResult returns the instance of the result specified by ID
+func (d *datastoreImpl) GetComplianceCheckResult(ctx context.Context, complianceResultID string) (*storage.ComplianceOperatorCheckResultV2, bool, error) {
+	return d.store.Get(ctx, complianceResultID)
```

Comment:
we should check for read access here
if ok, err := complianceOperatorSAC.ReadAllowed(ctx); err != nil {
		return nil, false,err
	} else if !ok {
		return nil ,false,sac.ErrResourceAccessDenied
	}

Reply:
That is checked by the store